runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -949,7 +949,7 @@ void chpl_comm_broadcast_private(int id, size_t size, int32_t tid) {
                                           0, 0);
   if (payloadSize <= gasnet_AMMaxMedium()) {
     priv_bcast_t* pbp = chpl_mem_allocMany(1, payloadSize, CHPL_RT_MD_COMM_PRV_BCAST_DATA, 0, 0);
-    chpl_memcpy(pbp->data, chpl_private_broadcast_table[id], size);
+    chpl_memmove(pbp->data, chpl_private_broadcast_table[id], size);
```

Comment:
This cannot overlap; the memory `pbp` points to was allocated in the previous statement.